### DISCUSSION TITLE
Python3.11 Error while building with `make run`

### DISCUSSION THREAD

**Discussion by BernieHuang2008** [searxng/searxng#2735] [Q&A] [answered]
I'm currently building it with the github codespaces while some error occupied:
(a totally new environment works the same)
```
  ERROR: Failed building wheel for chompjs
  Running setup.py clean for chompjs
Successfully built cov-core
Failed to build chompjs
ERROR: Could not build wheels for chompjs, which is required to install pyproject.toml-based projects
ERROR: ./manage: line 352: error while pip install (local/py3/bin)
make: *** [Makefile:32: install] Error 42
```

However, if i change the `devcontainer.json` to use python3.8, all errors disappeared.
_**Do You Support Python 3.11 ???**_

**Comment by dalf** [comment, marked as answer]
Python 3.11 is supported but it seems the codespace configuration needs an update.
In the meantime run `sudo apt update; sudo apt install libpython3.11-dev` to fix the issue.

**Reply by BernieHuang2008** [reply]
thank you!
